forward_engineering/helpers/typeHelper.js

```diff
@@ -346,9 +346,45 @@ function getDiscriminator(discriminator) {
 	};
 }
 
+function correctType(value, type) {
```

Comment:
I would rename it to something like `parseExampleValueByDataType` to make it not look like it's related to the schema types handled in this file.
Also, it's better to place it in another file, you may create a new one if required, `exampleDataHelper` or smth like this.